stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -13,13 +13,15 @@
 import SwiftPrivate
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
-#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || CYGWIN
+#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin)
 import Glibc
 #endif
 
 
-#if !os(Windows) || CYGWIN
-// posix_spawn is not available on Android or Windows.
+#if os(Windows)
+// posix_spawn is not available on Windows.
+#else
```

Comment:
Better to collapse these two conditionals, I think. There's nothing particularly special about Windows or Android.